draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -1091,6 +1091,11 @@ In wire format, the value of the parameter is an ECHOConfigs vector
 {{!ECHO}}, including the redundant length prefix.  In presentation format,
 the value is encoded in {{!base64=RFC4648}}.
 
+The value of this parameter and any included ECHOConfig extensions are directed
+at the outer TLS ClientHello when ECHO is in use. This parameter does not affect
+parameters of the inner TLS ClientHello. Thus, changes targeting the inner TLS
+ClientHello when ECHO is in use SHOULD use other SVCB parameters.
```

Comment:
```suggestion
construction of the inner TLS ClientHello.
```
I'm suggesting we can do without this sentence entirely.

Reply:
Lovely -- thanks!